antenna/app.py

```diff
@@ -49,6 +49,9 @@ def filter(self, record):
             }
         },
         'formatters': {
+            'socorroapp': {
+                'format': '%(asctime)s %(levelname)s - %(name)s - %(message)s',
+            },
```

Comment:
This fixes logging formatting so when you're running it in the local dev environment, you can see which logger logged what.